src/sqlancer/postgres/PostgresToStringVisitor.java

```diff
@@ -362,4 +372,61 @@ public void visit(PostgresLikeOperation op) {
         super.visit((BinaryOperation<PostgresExpression>) op);
     }
 
+    @Override
+    public void visit(PostgresWindowFunction windowFunction) {
+        sb.append(windowFunction.getFunctionName());
+        sb.append("(");
+        // Fix: Use visitList instead of visit for a list of expressions
+        visitList(windowFunction.getArguments());
+        sb.append(") OVER (");
+
+        WindowSpecification spec = windowFunction.getWindowSpec();
+        if (!spec.getPartitionBy().isEmpty()) {
+            sb.append("PARTITION BY ");
+            visitList(spec.getPartitionBy());
+        }
+
+        if (!spec.getOrderBy().isEmpty()) {
+            if (!spec.getPartitionBy().isEmpty()) {
+                sb.append(" ");
+            }
+            sb.append("ORDER BY ");
+            // Fix: Create a method to handle order by terms specifically
+            visitOrderByList(spec.getOrderBy());
+        }
+
+        if (spec.getFrame() != null) {
+            sb.append(" ");
+            WindowFrame frame = spec.getFrame();
+            sb.append(frame.getType().getSQL());
+            sb.append(" BETWEEN ");
+            visit(frame.getStartExpr());
+            sb.append(" AND ");
+            visit(frame.getEndExpr());
+        }
+
+        sb.append(")");
+    }
+
+    // this method handles lists of expressions
+    private void visitList(List<PostgresExpression> expressions) {
+        int i = 0;
+        for (PostgresExpression expr : expressions) {
+            if (i++ != 0) {
+                sb.append(", ");
+            }
+            visit(expr);
+        }
+    }
+
+    // this method handles lists of order by terms
+    private void visitOrderByList(List<PostgresOrderByTerm> orderBy) {
```

Comment:
`PostgresOrderByTerm` is a `PostgresExpression`, so I think we don't need a separate method here.